public/index.html

```diff
@@ -10,6 +10,20 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!--  -->
```

Comment:
Could add something to this comment or maybe remove it entirely.

Reply:
Removed it .